universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -243,6 +279,52 @@ private DomContent formQuestionTypeSelect(QuestionType selectedType) {
         .withClasses(Styles.HIDDEN);
   }
 
+  /**
+   * Generate a {@link SelectWithLabel} fixed repeater selector based on the question definition's
+   * repeater.
+   */
+  private SelectWithLabel repeaterOptionWithQuestionDefinition(
+      QuestionDefinition questionDefinition) {
+    Path parentPath = questionDefinition.getPath().parentPath();
+    SimpleEntry<String, String> repeaterPathAndId =
+        new SimpleEntry<>(
+            parentPath.isEmpty() ? NO_REPEATER_DISPLAY_STRING : parentPath.toString(),
+            questionDefinition.getRepeaterId().map(String::valueOf).orElse(NO_REPEATER_ID_STRING));
+    return repeaterOptions(ImmutableList.of(repeaterPathAndId), "not used");
+  }
+
+  /**
+   * Generate a {@link SelectWithLabel} repeater selector of the available repeater question
+   * definitions.
+   */
+  private SelectWithLabel repeaterOptionsWithRepeaterQuestions(
+      QuestionForm questionForm,
+      ImmutableList<RepeaterQuestionDefinition> repeaterQuestionDefinitions) {
+    ImmutableList.Builder<SimpleEntry<String, String>> optionsBuilder = ImmutableList.builder();
+    optionsBuilder.add(new SimpleEntry<>(NO_REPEATER_DISPLAY_STRING, NO_REPEATER_ID_STRING));
+    optionsBuilder.addAll(
+        repeaterQuestionDefinitions.stream()
+            .map(
+                repeaterQuestionDefinition ->
+                    new SimpleEntry<>(
+                        repeaterQuestionDefinition.getPath().toString(),
```

Comment:
Suggestion (blocking): use the question name rather than the path for the option display value. Rationale is that we don't expose the path anywhere else to the admin so it's unlikely they'll have any means of interpreting it if we display it here.

Reply:
Agreed that it would be confusing to admin.

I'm conflicted about using just the name. The name may not be clear for repeated entities that exist with different contexts (e.g. applicant's jobs vs jobs for each household member). It also won't sufficiently describe nested repeated entities very clearly.

Reply:
Could we do something like "applicant -> child -> name"? Maybe something more sophisticated than arrows, but some indication of the chain of references

Reply:
i'm just doing `name` for now. we can make this fancier in the future.